extension/src/extension.ts

```diff
@@ -432,7 +446,35 @@ export class Extension extends Disposable implements IExtension {
 let extension: undefined | Extension
 export function activate(context: ExtensionContext): void {
   extension = new Extension(context)
-  context.subscriptions.push(extension)
+  const dvcYamlSelector = {
+    language: 'yaml'
+  }
+
+  context.subscriptions.push(
+    extension,
+    languages.registerDocumentLinkProvider(
```

Comment:
[Q] Does this return a disposable? Can we use the normal `dispose.track` pattern?

Reply:
Yes, this was the fastest way to get it working, but I'm currently integrating this with the Extension class like the other modules